### PR TITLE
Explain what the `Hush-Hush (Expert)` category means in `Medals`

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Medals/en.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ All Mod Introduction medals are eligible to be unlocked in any [game mode](/wiki
 
 As the Hush-Hush medals are meant to be secret achievements of some sort, the unlock requirements are not directly specified; however, players are [welcome to work together in order to figure out how to obtain all the Hush-Hush medals out there](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
 
+Hush-Hush also has its sub-category called "Hush-Hush (Expert)" the difference of which is medals in it don't allow any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods usage.
```
Comment (Walavouchey) [inline wiki/Medals/en.md:39]:
```suggestion
Hush-Hush also has its sub-category called "Hush-Hush (Expert)". The difference is that medals in it do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Comment by peppy** [comment]
Seems pretty arbitrary to have this separation for conditions which are also applied on many other medals.

**Comment by chromb** [comment]
> Seems pretty arbitrary to have this separation for conditions which are also applied on many other medals.

From what i know, all of the hush-hush medals that have that requirement are there, can't think of any exceptions off the top of my head.

**Comment by peppy** [comment]
I meant non-hush-hush cases.

**Comment by Walavouchey** [comment]
this just explains what the seperation for "expert" hush-hush medals on the site means. for other medals, whether difficulty reduction mods are allowed is indicated by their descriptions on the site

**Pull request merged**